project_proposal/tex/mainpage.tex

```diff
@@ -183,6 +183,9 @@ \section{Stakeholders}
 which features (user stories) that they would like developed next. Since such a
 person does not exist for this project, the group have instead pinpointed
 Kristian Nymann Jakobsen to fulfill this role.
+The group do not actually have any stakeholder because they don't have any 
```

Comment:
Maybe this bit should be moved to the start of this section. Write something like:  the group doesn't have the opportunity to communicate with Reflevsbæk, and therefore Kristian has been elected as PO.

Reply:
that sound good! 
I will try that thanks!